<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>80</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Century Gothic</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Nile</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="viewsSW">
      <property name="font">
       <font>
        <family>Century Gothic</family>
       </font>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="loginPage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QFrame" name="loginFrame">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <family>Century Gothic</family>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QFormLayout" name="formLayout_3">
           <property name="labelAlignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="formAlignment">
            <set>Qt::AlignCenter</set>
           </property>
           <item row="1" column="0" colspan="2">
            <widget class="QGroupBox" name="newUserGb">
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Century Gothic</family>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <property name="title">
              <string>New to Nile?</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="1" column="0">
               <widget class="QLabel" name="newUserLabel">
                <property name="font">
                 <font>
                  <family>MS Shell Dlg 2</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>User Name:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="newUserLE">
                <property name="font">
                 <font>
                  <family>MS Shell Dlg 2</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="emailLabel">
                <property name="font">
                 <font>
                  <family>MS Shell Dlg 2</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Email: </string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="emailLE">
                <property name="font">
                 <font>
                  <family>MS Shell Dlg 2</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="newPasswordLabel">
                <property name="font">
                 <font>
                  <family>MS Shell Dlg 2</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Password: </string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="newPasswordLE">
                <property name="font">
                 <font>
                  <family>MS Shell Dlg 2</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QPushButton" name="createPB">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="font">
                 <font>
                  <family>MS Shell Dlg 2</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="cursor">
                 <cursorShape>OpenHandCursor</cursorShape>
                </property>
                <property name="focusPolicy">
                 <enum>Qt::WheelFocus</enum>
                </property>
                <property name="text">
                 <string>Create</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QGroupBox" name="userGB">
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Century Gothic</family>
              </font>
             </property>
             <property name="title">
              <string>Login</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="LoginEmailLabel">
                <property name="font">
                 <font>
                  <family>Century Gothic</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Email:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="userLE">
                <property name="font">
                 <font>
                  <family>Century Gothic</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="passwordLabel">
                <property name="font">
                 <font>
                  <family>Century Gothic</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Pasword:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="paswordLE">
                <property name="font">
                 <font>
                  <family>Century Gothic</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QPushButton" name="loginButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="font">
                 <font>
                  <family>Century Gothic</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="cursor">
                 <cursorShape>OpenHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string>Login</string>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="nilePage">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QComboBox" name="categoriesCB">
          <item>
           <property name="text">
            <string>Todos los departamentos</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Alimentos y Bebidas</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Libros</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Electrónicos</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hogar Y Cocina</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Deporte y Aire Libre</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="searchLE"/>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="orderFCB">
          <item>
           <property name="text">
            <string>Sin filtro</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Menor a mayor $</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mayor a menor $</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="carPB">
          <property name="text">
           <string>Carrito de Compra</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="LogoutPB">
          <property name="text">
           <string>Cerrar Sesion</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QScrollArea" name="stuffSA">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="stuffContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>633</width>
             <height>610</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QGridLayout" name="productsLO"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="1" column="2" colspan="3">
         <widget class="QScrollArea" name="stuffSA_2">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="stuffContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>321</width>
             <height>610</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <layout class="QGridLayout" name="productsLO_2"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
